src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/BaseOptions.java

```diff
@@ -1045,6 +1094,14 @@ public int sourceTabSize() {
         return sourceTabSize;
     }
 
+    /**
+     * Argument for command-line option {@code --spec-base-URI}.
```

Comment:
I think the option is lower-cased `--spec-base-uri`.

Reply:
yes, and now `--spec-base-url`.